components/compliance-service/reporting/relaxting/migrations.go

```diff
@@ -347,7 +347,7 @@ func (backend ES2Backend) getScanDateRange(indexPrefix string) (*time.Time, *tim
 		return nil, nil, errors.Wrap(err, fmt.Sprintf("%s unable to Unmarshal max_date", myName))
 	}
 
-	logrus.Debugf("GetMinScanDate: most recent scan date for range = %s", maxDateAsString)
```

Comment:
These two references to `GetMinScanDate` are probably some old copy-paste cruft.